cpu/stm32/include/vendor/stm32f103xe.h

```diff
@@ -1,15 +1,15 @@
 /**
   ******************************************************************************
-  * @file    stm32f101xe.h
+  * @file    stm32f103xe.h
   * @author  MCD Application Team
-  * @brief   CMSIS Cortex-M3 Device Peripheral Access Layer Header File. 
-  *          This file contains all the peripheral register's definitions, bits 
-  *          definitions and memory mapping for STM32F1xx devices.            
-  *            
+  * @brief   CMSIS Cortex-M3 Device Peripheral Access Layer Header File.
+  *          This file contains all the peripheral register's definitions, bits
+  *          definitions and memory mapping for STM32F1xx devices.
+  *
   *          This file contains:
   *           - Data structures and the address mapping for all peripherals
   *           - Peripheral's registers declarations and bits definition
-  *           - Macros to access peripheral's registers hardware
+  *           - Macros to access peripheral�s registers hardware
```

Comment:
Is this no ASCII character present upstream?

Reply:
I haven't checked in detail tbh. The file is taken directly from the cmsis as downloaded in master (with just trailing white spaces removed)

Reply:
Ok, why the change of labels? Should we merge already?

Reply:
> https://github.com/RIOT-OS/RIOT/pull/15046

I thought there was something wrong with the check labels actions. But it turns out that 2020.07 doesn't contain the check-labels action. I guess @miri64 knows how to manage that.